examples/flat-serialisation/flat-single-direct.json

```diff
@@ -0,0 +1,76 @@
+{
+  "statementGroups": [
+    {
+      "id": "57d881b9-a35c-413e-870f-39ab8a1ac9e2",
+      "version": "0.1",
+      "beneficialOwnershipStatements": [
+        {
+          "id": "fbfd0547-d0c6-4a00-b559-5c5e91c34f5c",
+          "entity": {
+            "type": "entityStatement",
+            "statementDate": "2017-11-18",
+            "id": "1dc0e987-5c57-4a1c-b3ad-61353b66a9b7"
```

Comment:
We should consider using `statementID` rather than just `id` to make clear that this links to another **statement** rather than an identified entity. 

The use of `id` is important to making use of standard  flatten tool and merge tooling, but I think (a) this is less of an issue if we move towards a complete cross-referenced statement model, and (b) I would anticipate we could always set tools to have a configuration option for this.